lib/DebuggerAgent.js

```diff
@@ -105,7 +105,7 @@ DebuggerAgent.prototype = {
     this._debuggerClient.request(
       'scripts',
       {
-        includeSource: true,
+        includeSource: false,
```

Comment:
See https://github.com/node-inspector/node-inspector/commit/39fe73b620738381efbb9db3239e6eba074de1cc and #307

If the new front-end does not need `_getSourceMapUrl` anymore, then it's safe to change this flag back to `false`. Otherwise it would be better to investigate this more.

Regardless of the solution, please consider making the change in a new pull request and explaining the context of the change. It will make our life easier later in the future, when we will look at this line again and scratch our head trying to understand why it was changed.